src/mui/detail/Show.js

```diff
@@ -71,34 +73,37 @@ export class Show extends Component {
         );
 
         return (
-            <div>
-                <Card
-                    className="aor-show-layout"
-                    style={{ opacity: isLoading ? 0.8 : 1 }}
-                >
-                    <div className="aor-show-actions">
-                        {actions &&
-                            React.cloneElement(actions, {
-                                basePath,
-                                data,
-                                hasDelete,
-                                hasEdit,
-                                refresh: this.refresh,
-                                resource,
-                            })}
-                    </div>
-                    <ViewTitle title={titleElement} />
-                    {data &&
-                        React.cloneElement(children, {
-                            resource,
-                            basePath,
-                            record: data,
-                            translate,
-                            version,
-                        })}
-                </Card>
+          <div style={{ opacity: isLoading ? 0.8 : 1 }}>
```

Comment:
Why excluding this specific style from defaultStyles.js?

Reply:
I removed this style in my next pr